src/index.html

```diff
@@ -649,6 +651,13 @@ <h1 class="clear">Map boundary change</h1>
       </div>
     </div>
     <div class="map">
+      <div class="w-embed">
+        <style>
+.map-location__tags {
+	direction: ltr;
+}
+</style>
```

Comment:
There is already a map-location__tags class in the CSS, could this be added to that class rather than in the HTML?